src/app.py

```diff
@@ -4,14 +4,21 @@
 import time
 import algorithm
 import constants
+import cache
 
-reddit = praw.Reddit('MemeAdviser')
+# check if this is being imported or not
+if __name__ not "__main__":
```

Comment:
```suggestion
if __name__ != "__main__":
```
`SyntaxError: invalid syntax`